content/technotes/2019-01-10-hugo.md

```diff
@@ -0,0 +1,109 @@
+
+```
+[taxonomies]
+  tag = "tags"
+  author = "authors"
+```
+
+We added this option as well
+
+```
+preserveTaxonomyNames = true
+```
+
+to not mess up with authors' names capitalization when listing them.
+
+For authors, a challenge was moving metadata around, from storing authors' metadata (online home, Twitter username) inside posts metadata to having authors metadata files under content/authors, see [this example](https://github.com/ropensci/roweb2/blob/master/content/authors/ignasi-bartomeus/_index.md), but as mentioned earlier, with the `yaml` package you can edit hundreds of Markdown files at once, granted you check the edits carefully.
+
+Read more about [Hugo taxonomies](https://gohugo.io/content-management/taxonomies/), how to write [templates for taxonomy pages](https://gohugo.io/templates/taxonomy-templates/) and [where to save them so Hugo finds them](https://gohugo.io/templates/lookup-order/#examples-layout-lookup-for-taxonomy-list-pages). In our case, the tags and authors pages have different templates: [tags page](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/terms.html), [tag pages](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/taxonomy/tag.html), [authors templates](https://github.com/ropensci/roweb2/tree/master/themes/ropensci/layouts/authors)
+
+### Archive
+
+With precious content scattered over three categories (blog, tech notes, community calls), getting an overview can be tricky hence our adding an [archive page](/archive) featuring all our website content [from our very first blog post](https://ropensci.org/blog/2012/11/26/is-invasive/). Behind the scenes, we [list pages with a date](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/single.html#L29) and [feeds them to an archive partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/archive.html) that we use for the [technotes page](/technotes) and authors pages as well for consistent presentation.
+
+And should you ever get lost, we've now got a [cool 404 page](/404), although mentioning it probably lessens its coolness. [Here is its source](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/404.html).
+
+## More content
+
+In the previous section we mention the [community calls living in ropensci.org](/commcalls). They used to have their own source and subdomain, but we've transferred them which has several advantages:
+
+* When reading about community calls, you're looking at the same design than when reading tech notes which makes us happy since we like our website. Besides, thanks to the footer and header, you're more prone to bouncing on over parts of the website. The [community calls list](/commcalls) list and [individual pages](https://ropensci.org/commcalls/2018-12-18/) use partials from the rest of the website, and our Hugo experience. In each community call page, the video recording is for instance embedded!
```

Comment:
"thanks to the footer and header, you’re more prone to bouncing on over parts of the website" could use some clarification. Do you mean "thanks to the footer and header, you’re more likely to bounce over to another part of the website, rather than leave" ??